hudi-flink/src/main/java/org/apache/hudi/operator/partitioner/BucketAssignFunction.java

```diff
@@ -100,6 +160,12 @@ public void initializeState(FunctionInitializationContext context) {
             TypeInformation.of(HoodieKey.class),
             TypeInformation.of(HoodieRecordLocation.class));
     indexState = context.getKeyedStateStore().getMapState(indexStateDesc);
+    MapStateDescriptor<String, Integer> partitionLoadStateDesc =
+        new MapStateDescriptor<>("partitionLoadState", Types.STRING, Types.INT);
+    partitionLoadState = context.getKeyedStateStore().getMapState(partitionLoadStateDesc);
```

Comment:
i remembered that `partitionLoadState` is a keystate rather than operatorstate, so can't access `initializeState` and `notifyCheckpointComplete`.